SwiftUIDZ/ContentView.swift

```diff
@@ -8,15 +8,82 @@
 import SwiftUI
 
 struct ContentView: View {
+    enum Constants {
+        static let oldText = "Старый текст"
+        static let newText = "Новый текст"
+        static let updateText = "Обновить текст"
+        static let showBlockOneText = "Показать Блок1"
+        static let showBlockTwoText = "Показать Блок2"
+        static let showBlockThreeText = "Показать Блок3"
+        static let showBlockFourText = "Показать Блок4"
+        static let alertOneTitleText = "Пример Alert"
+        static let alertTwoTitleText = "Пример Alert c 2 кнопками и логикой"
+        static let actionSheetOneTitleText = "Пример ActionSheet"
+        static let actionSheetTwoTitleText = "Пример ActionSheet c кнопками и логикой"
+    }
+    
+    @State var isBlockOne = false
+    @State var isBlockTwo = false
```

Comment:
```suggestion
    @State var  = false
    
```
Какое-нибудь более информативное название придумай